.github/workflows/GuideConverterAction.yaml

```
@@ -0,0 +1,100 @@
+name: GuideConverter
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch pushed to'     
+        required: true
+        default: 'defaulted'
```

```suggestion
        default: 'qa'
```

Add a more useful default.

---

done